python/triton/runtime/jit.py

```diff
@@ -280,22 +280,14 @@ def has_default(self):
 specialize_impl_cache = []
 
 
-def create_specialize_impl():
-    if specialize_impl_cache:
-        return specialize_impl_cache[-1]
+def create_specialize_impl(specialize_extra):
 
     from ..language import constexpr
 
-    def specialize_impl(arg, specialize_extra, is_const=False, specialize_value=True, align=True):
+    def specialize_impl(arg, is_const=False, specialize_value=True, align=True):
```

Comment:
do we still want the higher order function vs just doing `*, _constexpr=triton.language.constexpr` to bring the symbol into locals?